frameworks/Elixir/phoenix/web/controllers/page_controller.ex

```diff
@@ -5,17 +5,20 @@ defmodule Hello.PageController do
 
   def index(conn, _params) do
     conn
+    |> put_resp_content_type("application/json", nil)
```

Comment:
@josevalim `utf-8` added by default to all `content-type`. And I see warning from FB. Maybe we can ignore it. What do you think?